gem.md

```diff
@@ -135,11 +140,7 @@ Gem Semantics
     rule <k> Gem GEMID . burn ACCTSRC VALUE => . ... </k>
          <gem>
            <gem-id> GEMID </gem-id>
-           <gem-balances>
-             ...
-             ACCTSRC |-> ( BALANCE_SRC => BALANCE_SRC -Rat VALUE )
-             ...
-           </gem-balances>
+           <gem-balances> ... ACCTSRC |-> ( BALANCE_SRC => BALANCE_SRC -Rat VALUE ) ... </gem-balances>
            ...
          </gem>
       requires VALUE >=Rat 0
```

Comment:
Shouldn't we have a check here that the amount being burned is not more than the account's balance (I haven't actually checked the contract source)?

Reply:
Fixed.

Reply:
The `gem` contracts are third-party supplied, so this is just a model of a "generic" gem contract. So it makes sense to make it as "tight" as possible.